### PR TITLE
Fixed response, remove dotenv now environment varibles load from appl…

### PR DESCRIPTION
**Pull request by toptierlabs**
…ication.yml and example files added. Also changed readme

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -8,6 +8,8 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(:default, Rails.env)
 
+ENV.update YAML.load(File.read('config/application.yml'))[Rails.env] rescue {}
```
Comment (toptierlabs) [inline config/application.rb]: isn't better use (figaro)[https://github.com/laserlemon/figaro] ? because if you add circle ci, you can't upload this file

Comment (toptierlabs) [inline config/application.rb]: When you add circle ci you can set environment variables

Comment (MaicolBen) [inline config/application.rb]: Perfect, I thought that circle ci wouldn't be able to create this file

Comment (toptierlabs) [inline config/application.rb]: Perfect, I thought that circle ci wouldn't be able to create this file

**Pull request merged**